jarbas/api/tests/test_reimbursement_view.py

```diff
@@ -1,59 +1,19 @@
-from datetime import date
 from json import loads
 from unittest.mock import patch
+from urllib.parse import urlencode
 
-from django.core.cache import cache
 from django.shortcuts import resolve_url
 from django.test import TestCase
 
```

Comment:
Unnecessary line: mixer can be imported in the sabe block of Django import (third party imports, according to PEP8)